deploy/docker/README.md

```diff
@@ -21,24 +21,29 @@ DOCKER_BUILDKIT=1 docker build -f deploy/docker/Dockerfile -t lowcoderorg/lowcod
+| `FRONTEND_ENABLED`              | If **true** lowcoder web frontend is started in the container       | `true`                                                |
+| `PUID`                          | ID of user running services. It will own all created logs and data. | `9001`                                                |
+| `PGID`                          | ID of group of the user running services.                           | `9001`                                                |
 | `MONGODB_URI`                   | Mongo database connection string                                    | `mongodb://localhost:27017/lowcoder?authSource=admin` |
-| `REDIS_URL`                     | Redis server URL                                                    | `redis://localhost:6379`                                |
-| `JS_EXECUTOR_URI`               | Node service URL                                                    | `http://localhost:6060`                                 |
-| `ENABLE_USER_SIGN_UP`           | Enable registration of new users                                    | `true`                                                  |
+| `REDIS_URL`                     | Redis server URL                                                    | `redis://localhost:6379`                              |
+| `JS_EXECUTOR_URI`               | Node service URL                                                    | `http://localhost:6060`                               |
+| `ENABLE_USER_SIGN_UP`           | Enable registration of new users                                    | `true`                                                |
 | `ENCRYPTION_PASSWORD`           | Encryption password                                                 | `lowcoder.org`                                        |
 | `ENCRYPTION_SALT`               | Salt used for encrypting password                                   | `lowcoder.org`                                        |
-| `CORS_ALLOWED_DOMAINS`          | CORS allowed domains                                                | `*`                                                     |
-| `LOWCODER_API_SERVICE_URL`    | Lowcoder API service URL                                          | `http://localhost:8080`                                 |
-| `LOWCODER_NODE_SERVICE_URL`   | Lowcoder Node service (js executor) URL                           | `http://localhost:6060`                                 |
+| `CORS_ALLOWED_DOMAINS`          | CORS allowed domains                                                | `*`                                                   |
+| `LOWCODER_API_SERVICE_URL`      | Lowcoder API service URL                                            | `http://localhost:8080`                               |
+| `LOWCODER_NODE_SERVICE_URL`     | Lowcoder Node service (js executor) URL                             | `http://localhost:6060`                               |
```

Comment:
How is this different from "JS_EXECUTOR_URI"?

Reply:
Agreed. We'll keep LOWCODER_NODE_SERVICE_URL